test/test_van/test_bulkimport.py

```diff
@@ -19,8 +23,116 @@ def tearDown(self):
     @requests_mock.Mocker()
     def test_get_bulk_import_resources(self, m):
 
-        json = ['Contacts', 'Contributions', 'ActivistCodes', 'ContactsActivistCodes']
+        json = ['Contacts', 'Contributions',
```

Comment:
This will depend on the permissions the key has - it could also have RecurringCommitments for example.

Reply:
Definitely. 

I think that this works for now, as it is just mocking the expected schema, rather than a live connection.